bin/nyc.js

```diff
@@ -117,7 +117,7 @@ if (process.env.NYC_CWD) {
   } else if (argv._.length) {
     // wrap subprocesses and execute argv[1]
     var nyc = (new NYC())
-    nyc.cleanup()
+    nyc.reset()
```

Comment:
The issue with `all` was that it was calling `cleanup()`, which deleted directories, but did not recreate them.

> but maybe that guard should be in this file and not in the implementation.

I think I agree with you. I just wanted to avoid an additional change creeping into this PR.